src/libprojectM/Renderer/ShaderEngine.cpp

```diff
@@ -122,10 +122,63 @@ ShaderEngine::ShaderEngine()
     UNIFORM_V2F_C4F_VERTEX_POINT_SIZE = glGetUniformLocation(programID_v2f_c4f, "vertex_point_size");
     UNIFORM_V2F_C4F_T2F_VERTEX_TRANFORMATION = glGetUniformLocation(programID_v2f_c4f_t2f, "vertex_transformation");
     UNIFORM_V2F_C4F_T2F_FRAG_TEXTURE_SAMPLER = glGetUniformLocation(programID_v2f_c4f_t2f, "texture_sampler");
+
+
+
+	std::string line;
+	std::ifstream myfile(DATADIR_PATH "/shaders/projectM.cg");
```

Comment:
can we try not reading the shaders from files? it simplifies packaging. just stick them in header files if possible.